src/components/Proposals/ProposalPage/OPProposalPage/OPProposalOptimisticPage.jsx

```diff
@@ -1,12 +1,12 @@
-import { getDelegate } from "@/app/api/common/delegates/getDelegates";
-import { getDelegateStatement } from "@/app/api/common/delegateStatement/getDelegateStatement";
-import { getCurrentDelegators } from "@/app/api/common/delegations/getDelegations";
-import { getVotableSupply } from "@/app/api/common/votableSupply/getVotableSupply";
+import { fetchDelegate as apiFetchDelegate } from "@/app/api/common/delegates/getDelegates";
```

Comment:
Missing import for getDelegators

Reply:
great catch, thanks for the thorough review